unit-tests/live/d400/test-disparity-modulation.py

```diff
@@ -0,0 +1,52 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+# This test checks that A Factor of Disparity can be changed
+
+import pyrealsense2 as rs
+from rspy import test, log
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def test_amp_factor(am_device, input_factor_values: list):
+    """
+    This function set new A Factor value to advance mode device
+    :am_device: advance mode device
+    :input_factor_values: list of A Factor values
+    """
+    amp_factor = am_device.get_amp_factor()
+    output_factor_values = []
+
+    for factor_value in input_factor_values:
+        amp_factor.a_factor = factor_value
+        am_device.set_amp_factor(amp_factor)
+        output_factor_values.append(am_device.get_amp_factor().a_factor)
+
+    test.check_float_lists(input_factor_values, output_factor_values)
+
+
+device = test.find_first_device_or_exit()
+depth_sensor = device.first_depth_sensor()
```

Comment:
1. Why do I need `depth_sensor`?
2. Why do I need to close depth sensor if I didn't ask start streaming?